client.go

```diff
@@ -32,26 +32,26 @@ const (
 )
 
 // New returns a new Helm client with the provided options
-func New(options *Options) (*Client, error) {
+func (c *HelmClient) New(options *Options) (*HelmClient, error) {
```

Comment:
Wait, so now the `HelmClient` creates itself... with the `New` method? How do you instantiate the `HelmClient` on which you could call `New`? 🤔

Reply:
This "chicken-and-egg" problem should now be resolved :+1: